src/libs/actions/DemoActions.js

```diff
@@ -0,0 +1,284 @@
+                        },
+                    },
+                },
+                {
+                    onyxMethod: Onyx.METHOD.MERGE,
+                    key: `${ONYXKEYS.COLLECTION.REPORT}${expenseChatReportID}`,
+                    value: {
+                        pendingFields: {
+                            addWorkspaceRoom: null,
+                        },
+                        pendingAction: null,
+                    },
+                },
+                {
+                    onyxMethod: Onyx.METHOD.MERGE,
+                    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${expenseChatReportID}`,
+                    value: {
+                        [_.keys(expenseChatData)[0]]: {
+                            pendingAction: null,
+                        },
+                    },
+                },
+            ],
+            failureData: [
+                {
+                    onyxMethod: Onyx.METHOD.SET,
+                    key: `${ONYXKEYS.COLLECTION.POLICY_MEMBERS}${policyID}`,
+                    value: null,
+                },
+                {
+                    onyxMethod: Onyx.METHOD.SET,
+                    key: `${ONYXKEYS.COLLECTION.REPORT}${announceChatReportID}`,
+                    value: null,
+                },
+                {
+                    onyxMethod: Onyx.METHOD.SET,
+                    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${announceChatReportID}`,
+                    value: null,
+                },
+                {
+                    onyxMethod: Onyx.METHOD.SET,
+                    key: `${ONYXKEYS.COLLECTION.REPORT}${expenseChatReportID}`,
+                    value: null,
+                },
+                {
+                    onyxMethod: Onyx.METHOD.SET,
+                    key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${expenseChatReportID}`,
+                    value: null,
+                },
+            ],
+        },
+    );
+
+    // Navigate to the new workspace chat report
+    // We must call goBack() to remove the demo route from history
+    Navigation.goBack();
+    Navigation.navigate(ROUTES.getReportRoute(expenseChatReportID));
+}
+
+function runSbeDemo() {
+    // Add domain name as prefix (only if domain is private)
+    const domainNamePrefix = userIsFromPublicDomain ? `${Str.extractEmailDomain(sessionEmail)} ` : '';
+    const workspaceName = `${domainNamePrefix}SBE Workspace`;
+
+    const welcomeNoteText = `
+    Thanks for testing out New Expensify!
+    
+    Get paid back for your journey to or from Small Business Expo, whether you're driving yourself or catching a ride:
+    
+    1. Click the "+" > Request money > Distance.
+    2. Enter your start and end addresses.
+    3. Get paid back at the IRS mileage rate of $0.655/mile!
+    
+    Note: Must submit by Thursday, September 7th, 2023. One reimbursement per person.`;
+
+    createDemoWorkspaceAndNavigate(CONST.EMAIL.SBE, workspaceName, welcomeNoteText, CONST.ACCOUNT_ID.SBE, '');
```

Comment:
FYI we don't have an API command created for this yet... Do y'all prefer I comment this out so we can still move forward with this?